src/components/@core/WKT/search.tsx

```diff
@@ -85,28 +87,35 @@ export default function WKTSearch({
             loadOptions={onQueryDebounce}
           />
         </FormControl>
-        <Text color="gray.500" mb={2} children={t("common:or")} />
+        {gMapTab && <Text color="gray.500" mb={2} children={t("common:or")} />}
       </div>
-      <Box position="relative" borderRadius="md" mb={4}>
-        <NakshaGmapsDraw
-          defaultViewPort={defaultViewPort}
-          defaultFeatures={selected?.value ? [selected?.value] : []}
-          isControlled={true}
-          isAutocomplete={true}
-          isReadOnly={true}
-          onFeaturesChange={handleOnChange}
-          gmapRegion={SITE_CONFIG.MAP.COUNTRY}
-          gmapApiAccessToken={SITE_CONFIG.TOKENS.GMAP}
-          mapStyle={{ height: "22rem", width: "100%", borderRadius: ".25rem" }}
-          autocompleteComponent={
-            <FormControl mb={4}>
-              <FormLabel htmlFor="gmaps-search">{t("form:find_gmaps")}</FormLabel>
-              <Input ref={gmapsSearchRef} w="full" />
-              <FormHelperText>{t("form:coverage.hint")}</FormHelperText>
-            </FormControl>
-          }
-        />
-      </Box>
+      {!gMapTab ? (
```

Comment:
you can reverse internal elements instead of `!gMapTab` just simply use `gMapTab`